debian/control

```diff
@@ -6,11 +6,9 @@ Build-Depends: cmake (>= 2.8.11),
                debhelper (>= 9),
                dh-systemd,
                dpkg-dev (>= 1.17),
-               gcc,
                libnl-3-dev,
                libnl-route-3-dev,
                libudev-dev,
-               make,
                ninja-build,
```

Comment:
Benjamin,
I have following commit in our review system.
Is it correct one? If yes, can you squash it?

commit 0e6099884e72d1a6a35238884353778d0083fe40
Author: Talat Batheesh <talatb@mellanox.com>
Date:   Mon Dec 26 10:35:16 2016 +0200

    debian/control: Unmet build dependencies ninja
    
    Fixes  the following issues
    
        * debuild clean issue
        * dpkg-checkbuilddeps: error: Unmet build dependencies: ninja
    
    Signed-off-by: Talat Batheesh <talatb@mellanox.com>

diff --git a/debian/control b/debian/control
index 3b04ac9..367c30f 100644
--- a/debian/control
+++ b/debian/control
@@ -12,7 +12,7 @@ Build-Depends: build-essential,
               libnl-3-dev,
               libnl-route-3-dev,
               make,
-              ninja,
+              ninja-build,
               pkg-config,
               valgrind
 Standards-Version: 3.9.8
~